backend/README.md

```diff
@@ -24,10 +22,25 @@ $ npm install -g yarn && nodenv rehash
 $ yarn install
 ```
 
+## Android environment
+
+In order to use the Android verification endpoints you need to enable the Google Play Integrity API service on the Google Cloud project related to the `GOOGLE_CLOUD_PROJECT_NUMBER` provided in the example app `.env` file. Referer to the [setup page](https://developer.android.com/google/play/integrity/setup) for more information.
+
+Be sure to fill the `.env` file with the required enviroment variables:
+
+```javascript
+ANDROID_BUNDLE_IDENTIFIER =
+GOOGLE_APPLICATION_CREDENTIALS =
+```
+
+`ANDROID_BUNDLE_IDENTIFIER` is the package name of the app you want to verify, it's already included in the `.env.local` file for the example app of this project.
+
+`GOOGLE_APPLICATION_CREDENTIALS` consists of a JSON string with the service account credentials. Refer to the [instructions](https://developer.android.com/google/play/integrity/standard#decrypt-and) for more information.
+
 ## Run the app
 
 ```bash
-# First edit .env to add a BACKEND_ADDRESS (is required to use real local ip address instead localhost)
-$ cp .env.local env.local
+# First edit .env to add the required environment variables according to the platform you want to test
+$ cp .env.local .env
```

Comment:
What about explicit that once copied the `.env`, if we change values inside it we better do a **metro reset cache**?

Reply:
Addressed in the example app `README.md`, 428ad8842558fa29ff259210e5e41c82b42b1052, thanks.